openpype/hosts/max/plugins/publish/validate_loaded_plugin.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+"""Validator for Loaded Plugin."""
+import os
+from pyblish.api import InstancePlugin, ValidatorOrder
+from pymxs import runtime as rt
+
+from openpype.pipeline.publish import (
+    RepairAction,
+    OptionalPyblishPluginMixin,
+    PublishValidationError
+)
+from openpype.hosts.max.api.lib import get_plugins
+
+
+class ValidateLoadedPlugin(OptionalPyblishPluginMixin,
+                           InstancePlugin):
+    """Validates if the specific plugin is loaded in 3ds max.
+    Studio Admin(s) can add the plugins they want to check in validation
+    via studio defined project settings"""
+
+    order = ValidatorOrder
+    hosts = ["max"]
+    label = "Validate Loaded Plugins"
+    optional = True
+    actions = [RepairAction]
+
+    def get_invalid(self, instance):
+        """Plugin entry point."""
+        if not self.is_active(instance.data):
+            self.log.debug("Skipping Validate Loaded Plugin...")
+            return
+
+        required_plugins = (
+            instance.context.data["project_settings"]["max"]["publish"]
+                                 ["ValidateLoadedPlugin"]
+                                 ["family_plugins_mapping"]
+        )
+
+        if not required_plugins:
+            return
+
+        invalid = []
+
+        # get all DLL loaded plugins in Max and their plugin index
+        available_plugins = {
+            plugin_name.lower(): index for index, plugin_name in enumerate(
+                get_plugins())
+        }
+
+        for families, plugin in required_plugins.items():
+            families_list = families.split(",")
+            excluded_families = [family for family in families_list
+                                 if instance.data["family"] != family
+                                 and family != "_"]
+            if excluded_families:
+                self.log.debug("The {} instance is not part of {}.".format(
+                    instance.data["family"], excluded_families
+                ))
+                return
+
+            if not plugin:
+                return
+
+            plugin_name = plugin.format(**os.environ).lower()
+            plugin_index = available_plugins.get(plugin_name)
+
+            if plugin_index is None:
+                invalid.append(
+                    f"Plugin {plugin} not exists in 3dsMax Plugin List."
+                )
+                continue
+
+            if not rt.pluginManager.isPluginDllLoaded(plugin_index):
+                invalid.append(f"Plugin {plugin} not loaded.")
+
+        return invalid
+
+    def process(self, instance):
+        invalid_plugins = self.get_invalid(instance)
+        if invalid_plugins:
+            bullet_point_invalid_statement = "\n".join(
+                "- {}".format(invalid) for invalid in invalid_plugins
+            )
+            report = (
+                "Required plugins fails to load.\n\n"
```

Comment:
They don't necessarily _fail to load_ - they are just _not loaded_ right? Or how should I interpret this? 
```suggestion
                "Required plugins are not loaded.\n\n"
```